eng/pipelines/common/global-build-job.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osSubgroup: ''
+  container: ''
+  testGroup: ''
+  crossrootfsDir: ''
+  timeoutInMinutes: ''
+  signBinaries: false
+  publishToBlobFeed: false
+  stagedBuild: false
+  variables: {}
+  pool: ''
+
+jobs:
+- template: /eng/common/templates/job/job.yml
+  parameters:
+    name: ${{ format('runtime_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+    displayName: ${{ format('Runtime Build {0}{1} {2} {3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+    dependsOn: checkout
+    pool: ${{ parameters.pool }}
+    container: ${{ parameters.container }}
+
+    variables:
+    - name: configArg
+      value: "-c Release"
+    - ${{ if eq(parameters.buildConfig, 'debug') }}:
+      - name: configArg
+        value: "-c Debug"
```

Comment:
Could you just do this, or does capitalization matter?

```yaml
    - name: configArg
      value: '-c ${{ parameters.buildConfig }}'
```

Reply:
I think we should instead pass down as uppercase values into the template. When calling build.cmd/.sh in the root, we expect uppercase values because we just call MSBuild, not like src/coreclr/build.cmd/sh which actually expects lowercase arguments as `-release, -checked, -debug`.